### PR TITLE
Update documentation about MySQL 5.6 installation

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 1]
```diff
@@ -15,11 +15,37 @@ Base installation
...
 
 
+MySQL installation
+******************
+
+Since Akeneo PIM 1.6, the minimal MySQL version is MySQL 5.6. Ubuntu 16.04 default PHP version is MySQL 5.7.
+You have two possibilities:
+
+* Work with MySQL 5.6. Actually, the only supported version of PHP for Akeneo PIM is 5.6. You need to downgrade your version to MySQL 5.6.
+* Work with MySQL 5.7. MySQL 5.7 is not officialy supported but works as experimental mode if you disable the ONLY_FULL_GROUP_BY mode.
```
Comment (MarieMinasyan) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: works in experimental

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 2]
```diff
@@ -15,11 +15,37 @@ Base installation
...
+
+
+MySQL 5.6 (supported)
+^^^^^^^^^^^^^^^^^^^
```
Comment (MarieMinasyan) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Missing "^" ?

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 3]
```diff
@@ -15,11 +15,37 @@ Base installation
...
         $ sudo apt-get install apache2
-        $ sudo apt-get install mysql-server
         $ sudo a2enmod rewrite
         $ sudo service apache2 restart
 
 
+MySQL installation
+******************
+
+Since Akeneo PIM 1.6, the minimal MySQL version is MySQL 5.6. Ubuntu 16.04 default PHP version is MySQL 5.7.
```
Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Nope, we still support MySQL 5.5, as it is Debian 8 version

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 4]
```diff
@@ -15,11 +15,37 @@ Base installation
...
+You have two possibilities:
+
+* Work with MySQL 5.6. Actually, the only supported version of PHP for Akeneo PIM is 5.6. You need to downgrade your version to MySQL 5.6.
```
Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Same comment

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 5]
```diff
@@ -15,11 +15,37 @@ Base installation
...
         $ sudo apt-get install apache2
-        $ sudo apt-get install mysql-server
         $ sudo a2enmod rewrite
         $ sudo service apache2 restart
 
 
+MySQL installation
+******************
+
+Akeneo PIM supports MySQL from 5.1 to 5.6. Ubuntu 16.04 default PHP version is MySQL 5.7.
```
Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Ubuntu 16.04 default **MySQL** version

Comment (jmleroux) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: ouch, sorry :)

**Pull request closed without merging**